added misc icons

kruzuahz opened the pull request:
added bunch of icons, don't think we have a usecase for all of them but it's nice to have:
- DLC 
- NPC  (AI and trader)
- scraps
- ships (no airships)
- expeditions

only icons no tex changes

kruzuahz committed:
added misc icons

dotSp0T commented:
if we want to have resources as such in the repo, I would like to have at least a txt file or similar of used and (or at least) unused icons. Or a readme section, or sth along that idea

kruzuahz committed:
added readme file with unused icons. removed (unused) duplicates

kruzuahz committed:
removed unused tex files

kruzuahz committed:
fix readme headers, renamed unused icons

kruzuahz commented:
I've added a readme file in the icons folder which contains all unused icon filenames sorted by type. some new other were already there. I've assumed the new dlc icons as 'used' since we have a usecase for them. 

Also removed a few duplicate image files.
Also noticed there were 2 tex files that should have been removed in the previous PR. I've removed those as well.
